automobile/lidar-sensors.md

```diff
@@ -64,12 +67,51 @@ SickLms291 {
 }
 ```
 
-The `noise` field specifies the standard deviation of the gaussian noise in
+The `noise` field specifies the standard deviation of the gaussian depth noise in
 meters.
 
 The `resolution` field specifies the number of points returned per layer per
 scan.
 
+#### Sick LD-MRS
+
+The `Sick LD-MRS` is a 2 or 4 layers lidar with a range of 300 meters and a
+field of view of respectively 110 or 85 degrees.
```

Comment:
We should write one sentence per line and one line per sentence in the MD files.